index.js

```diff
@@ -0,0 +1,91 @@
+const express = require('express');
+const knex = require('knex')
+const session = require('express-session');
+const KnexSessionStore = require('connect-session-knex')(session)
+const knexConfig = require('./knexfile.js')
+const db = knex(knexConfig.development)
+const bcrypt = require('bcryptjs')
+const server = express();
+
+const sessionConfig = {
+  name: 'monkey',
+  secret: 'hello world',
+  cookie: {
+    maxAge: 1000 * 60 * 20 * 2,
```

Comment:
Proper usage of maxAge. Usually students make mistakes here and make them too short.